frontend/wallet/src/render/views/settings/DelegationSettings.re

```diff
@@ -61,9 +65,18 @@ let make = (~publicKey) => {
 
   let onChangeFee = value =>
     switch (value) {
-    | 0 => changeState(prev => {delegate: prev.delegate, fee: DefaultAmount})
+    | 0 =>
+      changeState(prev =>
+        {delegate: prev.delegate, fee: DefaultAmount, error: state.error}
```

Comment:
This seems a little strange -- why are we getting delegate from prev and error from state?
As a shorthand, if you need to change one value in the record and keep the others the same, you can use the [record spread](https://reasonml.github.io/docs/en/record#immutable-update) syntax:
`{...prev, fee: DefaultAmount}`

Same applies below.